src/org/zaproxy/zap/extension/viewstate/zap/utils/JSFViewState.java

```diff
@@ -0,0 +1,92 @@
+package org.zaproxy.zap.extension.viewstate.zap.utils;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.extension.encoder.Base64;
+
+/**
+ * THIS CODE IS FROM THE PROJECT LOCATED AT
+ * http://code.google.com/p/embeddednode/ AND THE RIGHT HAS BEEN GRANTED BY HIS
```

Comment:
Something doesn't seem right in this sentence, also googlecode.com us gone do you have an up-to-date reference, name, or email address?